composer.json

```diff
@@ -28,12 +28,12 @@
     "phpdocumentor/reflection-docblock": "^2.0@dev"
   },
   "require-dev": {
-    "scriptfusion/phpunit-immediate-exception-printer": "^1",
+    "scriptfusion/phpunit-immediate-exception-printer": "^2",
     "phpunit/phpunit": "^5.7"
   },
   "autoload": {
     "psr-4": {
-      "Axyr\\IDEAnnotator\\": "src/"
+      "SilverLeague\\IDEAnnotator\\": "src/"
```

Comment:
Doh! Also need to add a Tests autoloader

Reply:
We might consider making this an autoload-dev definition so the namespaces will not be registered in production, another failsafe?